src/systemtest/java/reposense/LocalRepoSystemTest.java

```diff
@@ -0,0 +1,71 @@
+package reposense;
+
+import static reposense.util.TestUtil.loadResource;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import reposense.model.RepoConfiguration;
+import reposense.model.RepoLocation;
+import reposense.util.FileUtil;
+import reposense.util.SystemTestUtil;
+import reposense.util.TestRepoCloner;
+
+public class LocalRepoSystemTest {
+
+    private static final String LOCAL_DIRECTORY_ONE_PARENT = "parent1";
+    private static final String LOCAL_DIRECTORY_TWO_PARENT = "parent2";
+    private static final String LOCAL_DIRECTORY_ONE = "parent1/test-repo";
+    private static final String LOCAL_DIRECTORY_TWO = "parent2/test-repo";
+
+    private static final String OUTPUT_DIRECTORY = "local-test";
+    private static final Path REPORT_DIRECTORY_PATH = Paths.get(OUTPUT_DIRECTORY, "reposense-report");
+
+
+    @BeforeAll
+    public static void setupLocalRepos() throws Exception {
+        TestRepoCloner.clone(new RepoConfiguration(new RepoLocation("https://github.com/reposense/testrepo-Alpha")),
+                Paths.get("."),
+                LOCAL_DIRECTORY_ONE);
+        TestRepoCloner.clone(new RepoConfiguration(new RepoLocation(LOCAL_DIRECTORY_ONE)),
+                Paths.get("."),
+                LOCAL_DIRECTORY_TWO);
+    }
+
+    @AfterEach
+    public void deleteReportDirectory() throws Exception {
+        FileUtil.deleteDirectory(OUTPUT_DIRECTORY);
+    }
+
+    @AfterAll
+    public static void deleteClonedLocalRepos() throws Exception {
+        FileUtil.deleteDirectory(LOCAL_DIRECTORY_ONE_PARENT);
+        FileUtil.deleteDirectory(LOCAL_DIRECTORY_TWO_PARENT);
+    }
+
+    @Test
+    public void testSameFinalDirectory() {
+        String cliInput = String.format("-r %s %s -s d1 -u 01/04/2022 -o local-test",
+                LOCAL_DIRECTORY_ONE, LOCAL_DIRECTORY_TWO);
```

Comment:
I'm not sure why the style for the args is different from that of lines 33-34 and 36-37.

Reply:
I'm not too sure what you mean. The above you have quoted for lines 53 and 54 is the actual system test simulating if we were running it from CLI. Lines 33-34 and 36-37 is setup for the test as we have to clone local repositories before we can use them for testing.

Reply:
What I meant to say is I was curious why it's
```
       Paths.get("."),
       LOCAL_DIRECTORY_ONE
```
in lines 33-34 and not `Paths.get("."), LOCAL_DIRECTORY_ONE` on the same line.

Or the other way round with respect to line 54

Reply:
Oh I see! I think it was more that they were different types of arguments, since `Paths.get(".")` is the working directory and `LOCAL_DIRECTORY_ONE` is the output directory. Whereas to me, for lines 53 and 54, the two local directories were just string substitutions being performed so I did not see a need to separate them. It's just the way I decided to code it and I have no strong preference one way or the other. Do you want me to change it?

Reply:
I personally prefer having as many variables as possible on one line.